benchmark/buffers/buffer-base64-encode.js

```diff
@@ -3,15 +3,15 @@ var common = require('../common.js');
 
 const bench = common.createBenchmark(main, {
   N: [64 * 1024 * 1024],
```

Comment:
Sorry, I meant *this* parameter should be called `len`. `n` should stay the same.